ui/src/views/PageNotFoundView.vue

```diff
@@ -21,11 +28,21 @@
 </template>
 
 <script>
-  export default { name: "ErrorView" };
+  export default { name: "ErrorView",
+  data: function() {
+    return {
+      imageWidth: "35%"
+    }
+  },
+  computed: {
+    variables() {
+      return require("@styles/variables.js");
+    },
+    mainPicture() {
```

Comment:
The issue with the SVG file is that it has the text "designed by x" which does not look nice. That's why I converted the SVG file into PNG and removed that text. I could not find a good SVG editor to do that correctly.

Reply:
Aha, I see. Alright, lets leave that as png for now.. I think it can be done with vector editor such as Corel or ai but it takes time, better to use png then